lib/generators/authentication/templates/rspec/requests/api/sessions_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Sessions", type: :request do
+  let(:user) { users(:lazaro_nixon) }
```

Comment:
If you put a subject here, the call will be with is_expected and that is so good to read, personally.

```suggestion
RSpec.describe "Sessions", type: :request do
  subject { response }
  
  let(:user) { users(:lazaro_nixon) }
```